packages/react/src/components/Storage/FileUploader/__tests__/FileUploader.test.tsx

```diff
@@ -134,12 +135,12 @@ describe('File Uploader', () => {
       {
         ...fileStatus,
         percentage: 100,
-        fileState: 'success' as const,
+        fileState: FileState.SUCCESS as const,
```

Comment:
Do we need to cast to `const` still? Same question for lines 143, 182, 205, 563, 569, 592, 598, 670, 689 and 694

Reply:
I'll remove those

Reply:
[40f29e4](https://github.com/aws-amplify/amplify-ui/pull/3396/commits/40f29e47d4d31ab222b733e94b2f45e4b75f2a0f)